### ISSUE TITLE
FEAT: Add Links Option in the ChatInput Box

### ISSUE THREAD

**Issue by r3yc0n1c** [RocketChat/EmbeddedChat#678]
As an EmbeddedChat developer

**I need to**:
Implement the 'Add links' option in the ChatInput box

**So That**:
Users can easily insert hyperlinks into their messages, enhancing the functionality and user experience of the chat interface.

**Acceptance Criteria**

- [x] An icon for 'Add Links' is available in the ChatInput box.
- [x] Clicking the 'Add Links' button opens a modal or prompt for users to input the link text and URL.
- [x]  The inputted link is correctly formatted and added to the message, ensuring it's clickable and visually distinct.

**Demo**
![image](https://github.com/user-attachments/assets/45bbe9d5-58a4-47ba-913e-bfed19df4690)

Just like RC main has:
![image](https://github.com/user-attachments/assets/869ddca3-0a0c-4dae-9176-80809a8f3f9f)

**Comment by devanshkansagra** [comment]
Hey @r3yc0n1c, I am working on this issue.

**Comment by r3yc0n1c** [comment]
> Hey @r3yc0n1c, I am working on this issue.

was this an open issue from before?

**Comment by devanshkansagra** [comment]
No, I mean currently I found an approach so I said that I am working on this issue

**Comment by r3yc0n1c** [comment]
I was working on this when I noticed this was missing, will drop a PR now... I have some more issues to be listed... you can take one of those then?

**Comment by devanshkansagra** [comment]
Okay then, Ping me on Rocketchat id: devansh.kansagra

**Comment by r3yc0n1c** [comment]
sure

**Comment by Emankazmi9** [comment]
Is this issue still open, if yes could you please assign it to me?

**Comment by r3yc0n1c** [comment]
> Is this issue still open, if yes could you please assign it to me?

I'm working on it... was out of gitpod credits
